assets/css/_themes.scss

```
@@ -3,47 +3,41 @@
  ******************************************************************************/
 
 :root {
-  --global-bg-color: #{$grey-color-light};
-  --global-code-bg-color: #{$code-bg-color-light};
-  --global-text-color: #{$grey-color-dark};
-  --global-text-color-light: #{$grey-color};
-  --global-theme-color: #{$theme-light-color};
-  --global-hover-color: #{$theme-light-color-hover};
-  --global-footer-bg-color: #{$grey-color-light};
-  --global-footer-text-color: #{grey-color-dark};
-  --global-footer-link-color: #{$theme-light-color};
-  --global-distill-app-color: #{$grey-color-dark};
-  --global-divider-color: #{$grey-color};
-  --global-card-bg-color: #{$white-color};
+    --global-text-color-primary: #{$lightmode-text-color};
+    --global-text-color-secondary: #{$lightmode-text-color-light};
```

The naming convention is hard to follow for the end user. We are not using a secondary color here. We are using a shade of primary color. It should be named something like `text-color-primary`, `text-color-primary-shade`. 

The secondary color definition is very different from what we have used. For example, they use blue as primary and green as secondary. Also, they would pick different shades of these primary and secondary colors. 

 I actually talked to a UI/UX designer and asked for her idea. She said we only need a primary color for our website now, and the secondary color is unnecessary. Using this term would cause confusion. 

So, I suggest that we change the naming to like below. Also, I think we have to unify the naming convention. 

```suggestion
    --global-text-color-primary: #{$lightmode-text-color};
    --global-text-color-primary-shade: #{$lightmode-text-color-shade};
```